.github/workflows/ci.yaml

```diff
@@ -25,6 +25,15 @@ jobs:
         run: |
           make lint
           scripts/check-sync-tidy.sh
-
+      - name: Lint scripts
+        if: ${{ matrix.go-version == '1.21' }}
+        run: |
+          make lint-scripts
       - name: Test
-        run: make test
+        if: ${{ matrix.go-version != '1.18' }}
+        run: |
+          make test
```

Comment:
> the script uses library versions that are only available in go1.21, anyway that is the only Go version where its worth testing it because we will execute it from local machine

See comment above